packages/clay-modal/src/Hook.ts

```diff
@@ -82,7 +82,8 @@ const useUserInteractions = (
 		if (
 			elementRef.current &&
 			event.target !== null &&
-			!elementRef.current.contains(event.target as HTMLDivElement)
+			!elementRef.current.contains(event.target as HTMLDivElement) &&
+			document.contains(event.target as HTMLElement)
```

Comment:
This would always evaluate to `true` if the element exists in the DOM right?

Reply:
Yes, this ends up covering the case of when the element was removed from the DOM and is nowhere else, as in the case of #2165.